docs/application-settings.md

```diff
@@ -0,0 +1,298 @@
+p9        | Generate audience insights      | n\a
+p10      | Develop/improve products        | n\a
+
+f1         | Precise geo                               | Verifies user opt-in. If the user has opted out, rounds off the IP address and lat/long details 
+f2         | Fingerprinting                          | n\a
+```
+## File based
+
+In file based approach all configuration stores in .yaml files, path to which are defined in application properties.
+
+### Configuration in application.yaml
+
+```
+settings:
+  filesystem:
+    settings-filename: <directory to yaml file with settings>
+```
+### File format
+
+```
+accounts:
+  - id: 14062
+    bannerCacheTtl: 100
+    videoCacheTtl: 100
+    eventsEnabled: true
+    priceGranularity: low
+    enforceCcpa: true
+    analyticsSamplingFactor: 1
+    truncateTargetAttr: 40
+    gdpr:
+      enabled: true
+      purposes:
+        p1:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p2:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p3:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p4:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p5:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p6:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p7:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p8:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p9:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        p10:
+          enforce-purpose: full
+          enforce-vendors: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+      special-features:
+        sf1:
+          enforce: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+        sf2:
+          enforce: true
+          vendor-exceptions:
+            - bidder1
+            - bidder2
+      purpose-one-treatment-interpretation: ignore
+```
+
+  
+## Database application setting
+
+In database approach account properties are stored in database table with name accounts_account.
+
+### Configuration in application.yaml
+```
+settings:
+  enforce-valid-account: true
```

Comment:
Seems this section is not required here.
May be `circuit-breaker` is also under the question, but up to you.